modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -4,193 +4,57 @@ This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualB
 [source, bash]
 ----
-vboxmanage createmedium --filename "${VM_DISK}" --size 10240 --format VDI
-vboxmanage storageattach "${VM_NAME}" --storagectl SATA --type hdd --port 0 --device 0 --medium "${VM_DISK}"
-----
-
-[NOTE]
-====
-*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical reasons, disk images for some platforms ship with a smaller root filesystem, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown so that it is at least 8 GiB.
-
-Currently, if the root filesystem is smaller than 8 GiB, a warning is emitted on login. Starting from June 2021, if the root filesystem is smaller than 8 GiB and is followed by another partition, Fedora CoreOS will refuse to boot. For more details, see https://github.com/coreos/fedora-coreos-tracker/issues/586[this bug].
-====
-
-- Attach a DVD drive and the live ISO to the VM:
-
-[source,bash,subs="attributes"]
-----
-FCOS_DISK=$(realpath ~/Downloads/fedora-coreos-{stable-version}-live.x86_64.iso)
+STREAM="stable"
 
-vboxmanage storagectl "$\{VM_NAME}" --name IDE --add ide --controller PIIX4 --hostiocache on --portcount 2 --bootable on
-vboxmanage storageattach "$\{VM_NAME}" --storagectl IDE --type dvddrive --port 1 --device 0 --medium "$\{FCOS_DISK}"
+mkdir ova-templates
+coreos-installer download -s "${STREAM}" -p virtualbox -f ova -C ./ova-templates/
 ----
 
-You can now boot the target system you have configured. The ISO is capable of bringing up a fully functioning FCOS system purely from memory (i.e. without using any disk storage). Once booted, you will have access to a bash command prompt.
+Alternatively, OVA images can be manually downloaded from the https://getfedora.org/coreos/download/[download page].
 
-You can now run `coreos-installer` inside the VM:
-[source, bash]
-----
-sudo coreos-installer install /dev/sda \
-    --insecure-ignition --ignition-url http://10.0.2.2:8000/example.ign
-----
+== Booting a new VM on VirtualBox
 
-TIP: Check out `coreos-installer install --help` for more options on how to install Fedora CoreOS.
+This section shows how to use VirtualBox to configure and run VMs from thee CLI. Alternatively the graphical UI can be used as well.
 
-Once the installation is complete, you have to remove the live ISO after shutting down (`sudo shutdown -h now`) the VM. Once the ISO is removed, you can start the VM anew. The actual first boot process begins now. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
+=== Importing the OVA
 
+You can set up a VirtualBox virtual machine through the GUI or via the https://www.virtualbox.org/manual/UserManual.html#vboxmanage[`vboxmanage` CLI]. Below is a shell script that does most of what the graphical wizard will do. Note that this is tailored to a ova import. 
```

Comment:
"Most of"?  What does the GUI do that `vboxmanage` doesn't?

At this point it's not really a script; let's just call it a command.

Since we only provide an OVA image, I think the last sentence doesn't really add anything?

Reply:
Admittedly I added 'most of' from my own ignorance. I could find documentation on how to set some guest properties without the CLI. I kept coming across documentation that pointed me back to CLI. https://docs.oracle.com/en/virtualization/virtualbox/6.0/user/vboxmanage-guestproperty.html . While I assumed I was not finding the correct documentation / googlefooing I figured I would cover my bases with 'most of'.

That is vary fair, will do.